app/services/doc_auth/lexis_nexis/responses/lexis_nexis_response.rb

```diff
@@ -82,7 +82,10 @@ def extract_details(product)
           product[:ParameterDetails].each do |detail|
             group = detail[:Group]
             detail_name = detail[:Name]
-            value = detail.dig(:Values, 0, :Value)
+            is_region = detail_name.end_with?('Regions', 'Regions_Reference')
+            value = is_region ? detail.dig(:Values).map do |v|
+                                  v.dig(:Value)
+                                end : detail.dig(:Values, 0, :Value)
```

Comment:
I feel like `.dig` is really only useful with nested hashes, etc, because `nil.dig(:a)` will error. I think we could make this a tad more readable with:
```suggestion
            value = is_region ?
                detail[:Values].map { |v| v[:Value] } :
                detail.dig(:Values, 0, :Value)
```

Reply:
@zachmargolis fixed.